backend/src/Controllers/elections.controllers.ts

```diff
@@ -89,128 +93,120 @@ const getElectionResults = async (req: any, res: any, next: any) => {
 
 }
 const getSandboxResults = async (req: any, res: any, next: any) => {
-    console.log(`-> elections.getSandboxResults`)
+    Logger.info(req, `${className}.getSandboxResults`);
     try {
-        const candidateNames = req.body.candidates
-        const cvr = req.body.cvr
-        const num_winners = req.body.num_winners
-        console.log(candidateNames)
+        const candidateNames = req.body.candidates;
+        const cvr = req.body.cvr;
+        const num_winners = req.body.num_winners;
+        Logger.debug(req, "Candidate names:  "+JSON.stringify(candidateNames));
         const results = StarResults(candidateNames, cvr, num_winners)
-
         res.json(
             {
                 Results: results
             }
-        )
-    } catch (err) {
-        return res.status('400').json({
-            error: "Error calculating results"
-        })
+        );
+    } catch (err:any) {
+        Logger.error(req, `${className}.getSandboxResults error: ${err.message}.`);
+        return responseErr(res, req, 400, "Error calculating results");
     }
 }
 
 const getElections = async (req: any, res: any, next: any) => {
-    console.log(`-> elections.getElections`)
+    Logger.info(req, `${className}.getElections`);
+    var failMsg = "Could not retrieve elections";
     try {
         var filter = (req.query.filter == undefined) ? "" : req.query.filter;
         const Elections = await ElectionsModel.getElections(filter);
-        if (!Elections)
-            return res.status('400').json({
-                error: "Elections not found"
-            })
-        res.json(Elections)
-    } catch (err) {
-        return res.status('400').json({
-            error: "Could not retrieve elections"
-        })
+        if (!Elections){
+            Logger.error(req, failMsg);
+            return responseErr(res, req, 400, failMsg);
+        }
+        res.json(Elections);
+    } catch (err:any) {
+        Logger.error(req, failMsg+". "+err.message);
+        return responseErr(res, req, 400, failMsg);
```

Comment:
Maybe we can make this a 500 error? I think it only happens if something goes wrong on our end